README.md

```diff
@@ -182,26 +106,65 @@ Date.compare(start_date, date) != :gt and
   Date.compare(date, end_date) != :gt
 ```
 
-This way your code is more readable while still remaining correct.
+Your code is more readable while remaining correct!
 
-`CompareChain.compare?/1` is also available in case you only need chained comparison using the structural operators:
+`CompareChain.compare?/1` also enables chained comparison using the structural operators:
 
 ```elixir
 compare?(1 < 2 < 3)
 ```
 
-Though I find this case comes up less often.
+## Installation
+
+Add `compare_chain` to your project's dependencies in `mix.exs` and run `mix deps.get`:
+
+```elixir
+def deps do
+  [
+    {:compare_chain, "~> 0.5"}
+  ]
+end
+```
 
-### One last selling point
+## Usage
```

Comment:
You very rightly identified something I wasn't entirely sure how to best handle as I was reworking the doc.

My/our typical approach to structuring a README for an open source library like this is something like:

1. Tagline ("Chained semantic comparisons for Elixir." in our case here) and Badges
2. Key Features (if appropriate)
3. Installation
4. Usage
5. Contributing/Developing/Bug Reporting/etc. (if appropriate, though `CONTRIBUTING.md` is a better place for that, anyway)
6. ???
7. License

The "Background" section does a wonderful job explaining the value of the library but it is quite long. I absolutely didn't want to take a hatchet to it since it's well-written and is instructive and effective.

But, to your point, I agree that it really pushes installation and usage pretty far down the page. I'm comfortable reordering the three sections to:

1. Installation
2. Usage
3. Background

…if you agree that that makes sense.

So, I suppose the "why" in your original question is basically: I was attempting to thread a needle and I'm not sure I did it. 😄

Reply:
Yes I like that order 👍🏻 

I also like:

1. Usage
2. Installation
3. Background

since Usage is the window dressing. But I think Installation is more commonly above Usage so I'm fine with either.

Reply:
Looking [at the preview](https://github.com/CargoSense/compare_chain/blob/1b36d74f651f770039a33cb64a485b004873cf0a/README.md), this is much better now!